### ISSUE TITLE
Negative Electricity price limit not allowed

### ISSUE THREAD

**Issue by janneho** [jonasbkarlsson/ev_smart_charging#175]
## Version of the custom_component

Version 1.8.0

## Describe the bug
I have automation that sets Electricity price limit to last known price tomorrow.
Tomorrows price is -0,12c/kWh. It is not permitted to use negative price as limit
```
Failed to call service number/set_value. Value -0.12 for number.ev_smart_charging_electricity_price_limit is outside valid range 0.0 - 10000.0
```

Is it possible to change range to something like -10000.0 - 10000? No harm for anyone there if negative numbers are permitted.

**Comment by jonasbkarlsson** [comment]
That should be possible. Need to check that it doesn't cause any side-effects.

**Comment by jonasbkarlsson** [comment]
Sorry for slow response, but now this is included in [v1.9.0-dev2](https://github.com/jonasbkarlsson/ev_smart_charging/releases/tag/v1.9.0-dev2).

**Comment by jonasbkarlsson** [comment]
Part of release [v1.9.0](https://github.com/jonasbkarlsson/ev_smart_charging/releases/tag/v1.9.0).
